### PR TITLE
Finished POM changes

### PR DESCRIPTION
**Pull request by alexca1703**

### CODE REVIEW

File: cypress/page/checkout-information.page.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    public enterUserInformation(): void {
+        cy.get(this.firstNameTextField).type(this.firstNameValue);// (7)
+        cy.get(this.lastNameTextField).type(this.lastNameValue);// (8)
+        cy.get(this.zipCodeTextField).type(this.zipCodeValue);// (9)
+        cy.get(this.continueCheckoutButton).click();// (10)
+    }
+}
+
+export {SauceCheckoutInformation}
```
Comment (JuanDelahozMiranda) [inline cypress/page/checkout-information.page.ts]: Review the EOF error (add a blank line end of files)

**Review by JuanDelahozMiranda — CHANGES_REQUESTED** [review]

File: cypress/page/checkout-information.page.ts (hunk 1 above)
Comment (alexca1703) [reply cypress/page/checkout-information.page.ts]: added EOF

**Review by JuanDelahozMiranda — APPROVED** [review]

File: cypress/e2e/buy-shirt.cy.ts [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-        "Thank you for your order!",
-    ); // (12)
+    menuContentPage.visitMenuContentPage(); // (1)
+    LoginPage.enterStandardUserCredentials(); // (2)
+    InventoryPage.openProductDetails();// (3)
+    InventoryDetailsPage.addProductToCart(); // (4)(5)
+    ShoppingCartPage.proceedToCheckout(); // (6)
+    CheckoutInformationPage.enterUserInformation(); // (7)(8)(9)(10)
+    CheckoutOverviewPage.acceptCheckoutDetails(); // (11)
+    CheckoutCompletePage.confirmSuccessfulCheckout(); // (12)
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: please pass as a input parameter the message to be verified by confirmSuccessfulCheckout method

File: cypress/e2e/buy-shirt.cy.ts [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
-    cy.get("#last-name").type("Workshop");// (8)
-    cy.get("#postal-code").type("00000");// (9)
-    cy.get("input[id='continue']").click();// (10)
-    cy.get("button[id='finish']").click();// (11)
-    cy.get("#contents_wrapper > .checkout_complete_container > h2").should(
-        "have.text",
-        "Thank you for your order!",
-    ); // (12)
+    menuContentPage.visitMenuContentPage(); // (1)
+    LoginPage.enterStandardUserCredentials(); // (2)
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: please pass user and password as input parameters of this method. also, method should be called as login or signIn

File: cypress/e2e/buy-shirt.cy.ts [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
+  CheckoutOverview, FinishCheckout} from "../page/index";
+
+const menuContentPage = new SauceContentHomePage();
+const LoginPage = new SauceStandardLogin();
+const InventoryPage = new SauceInventory();
+const InventoryDetailsPage = new AddProductToCart();
+const ShoppingCartPage = new SauceCart();
+const CheckoutInformationPage = new SauceCheckoutInformation();
+const CheckoutOverviewPage = new CheckoutOverview();
+const CheckoutCompletePage = new FinishCheckout();
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: variables should be named in camelCase format. So instead of LoginPage you should use loginPage, and so on..

**Review by holgiosalos — CHANGES_REQUESTED** [review]

File: cypress/e2e/buy-shirt.cy.ts (hunk 2 above)
Comment (alexca1703) [reply cypress/e2e/buy-shirt.cy.ts]: Already fixed in the latest commit. Please check @holgiosalos @JuanDelahozMiranda

File: cypress/e2e/buy-shirt.cy.ts (hunk 3 above)
Comment (alexca1703) [reply cypress/e2e/buy-shirt.cy.ts]: Fixed in the latest commit, please check @holgiosalos @JuanDelahozMiranda

File: cypress/e2e/buy-shirt.cy.ts (hunk 4 above)
Comment (alexca1703) [reply cypress/e2e/buy-shirt.cy.ts]: Fixed in the latest commit, please check @holgiosalos @JuanDelahozMiranda

File: cypress/page/complete-checkout.page.ts [hunk 5]
```diff
@@ -1,15 +1,15 @@
 class FinishCheckout{
 
-    private condition: string;
-    private expectedText: string;
+    private messageElement: string;
+    //private condition: string;
```
Comment (holgiosalos) [inline cypress/page/complete-checkout.page.ts]: remove commented lines

File: cypress/page/complete-checkout.page.ts [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
 
     constructor() {
-        this.condition = "have.text";
-        this.expectedText = "Thank you for your order!";
+        this.messageElement = "#contents_wrapper > .checkout_complete_container > h2";
+        //this.condition = "have.text";
```
Comment (holgiosalos) [inline cypress/page/complete-checkout.page.ts]: remove commented lines

File: cypress/page/complete-checkout.page.ts (hunk 5 above)
Comment (alexca1703) [reply cypress/page/complete-checkout.page.ts]: Done, please review @holgiosalos

File: cypress/page/complete-checkout.page.ts (hunk 6 above)
Comment (alexca1703) [reply cypress/page/complete-checkout.page.ts]: Done, please review @holgiosalos

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**